.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: 'Report a bug for Dynatrace Service '
+title: ''
+labels: type:bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Versions**
+
+Versions of the applicable components
+
+* Dynatrace Service version : 
+* Dynatrace cluster version :
```

Comment:
maybe we could add the `Keptn` version as well here

Reply:
Perfect, thanks, will update the PR :)